lona/client/_lona/context.js

```diff
@@ -53,7 +53,7 @@ export class LonaContext {
             this.settings.scroll_to_top_on_view_start = true;
         };
 
-        this._windows = {};
+        this._windows = new Map();
```

Comment:
Looks line need to update `test_project/templates/lona/frontend.html` as well

Reply:
Thank's for catching that! I added a change to the patch for that.